src/js/components/Layer/stories/Form.js

```diff
@@ -91,6 +90,4 @@ const FormLayer = () => {
   );
 };
 
-storiesOf('Layer', module).add('Form', () => <FormLayer />, {
-  chromatic: { disable: true },
-});
+FormLayer.story = { name: 'Form' };
```

Comment:
Can you add the disabling of chromatic like it shows on the old code? You can see other PRs for example.

Reply:
I don't know how I missed that! Fixing it right now